File: 8KYU/Closest Elevator/solution.rb
def elevator(left, right, call)
    left_dist = (call - left).abs
    right_dist = (call - right).abs
    
    if left_dist == right_dist
        return "right"
    end
    if left_dist < right_dist
        return "left"
    end
    return "right"
end

File: 8KYU/Closest Elevator/solution.c
const char* elevator(int left, int right, int call) {

    int left_dist = abs(call - left);
    int right_dist = abs(call - right);
  
    if(left_dist == right_dist)
        return "right";
    else if(left_dist < right_dist)
        return "left";
    return "right";
}